common/app/components/Player.tsx

```diff
@@ -683,6 +683,31 @@ export default function Player({
         return () => clearInterval(interval);
     }, [updatePlaybackRate, subtitleCollection, clock, subtitles, playMode, settings.fastForwardModePlaybackRate]);
 
+    useEffect(() => {
+        if (playMode === PlayMode.repeat) {
+            autoPauseContext.onWillStopShowing = () => {
```

Comment:
I don't think this is causing any actual bugs right now but this particular `onWillStopShowing` assignment is overriding the assignment made on line 259. Can you move this logic into the existing subscription `handleOnWillStopShowingSubtitle`?